internal/expressions/json_templates.go

```diff
@@ -105,7 +105,13 @@ func EvaluateTemplate(template string, env map[string]any, exprOpts ...expr.Opti
 	}
 	exprOpts = append(exprOpts, expr.Function(
 		"quote",
-		func(a ...any) (any, error) { return fmt.Sprintf(`"%v"`, a[0]), nil },
+		func(a ...any) (any, error) {
+			jsonBytes, err := json.Marshal(a[0])
```

Comment:
Should we `json.Valid` before attempting to marshal?

Reply:
Discussed offline.